bin/exec-env

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env bash
 
-if [ "$MIX_ARCHIVES" = "" ]; then
-  export MIX_ARCHIVES=$ASDF_INSTALL_PATH/.mix/archives
-fi
-
 if [ "$MIX_HOME" = "" ]; then
   export MIX_HOME=$ASDF_INSTALL_PATH/.mix
 fi
 
+if [ "$MIX_ARCHIVES" = "" ]; then
+  export MIX_ARCHIVES=$MIX_HOME/archives
```

Comment:
Why this path change?

Reply:
Otherwise the MIX_ARCHIVES path does not respect MIX_HOME. MIX_HOME should take precedence over ASDF_INSTALL_PATH. So essentially this is not changing anything, it's just fixing if you pass MIX_HOME it's also used for archives.